js/main.js

```diff
@@ -117,9 +117,10 @@ function showPage (page) {
 
 function submitAmount (event) {
   event.preventDefault()
+  if (typeof(preSubmitAmount) !== 'undefined') preSubmitAmount(event)
```

Comment:
Not fond of the naming but I can't find a better proposition, I can live with it.